hudi-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataPayload.java

```diff
@@ -756,7 +756,7 @@ public HoodieRecordGlobalLocation getRecordGlobalLocation() {
     if (recordIndexMetadata.getFileIndex() != RECORD_INDEX_MISSING_FILEINDEX) {
       fileId += "-" + recordIndexMetadata.getFileIndex();
     }
-    final Date instantDate = new Date(recordIndexMetadata.getInstantTime());
+    final java.util.Date instantDate = new java.util.Date(recordIndexMetadata.getInstantTime());
```

Comment:
to be reverted

Reply:
This one was causing a test failure @xushiyan